canvas_manage_course/settings/isites_export_tool.py

```diff
@@ -1,15 +1,24 @@
 from .aws import *
 
 INSTALLED_APPS = (
+    'async',
```

Comment:
As discussed with @sapnamysore offline, the old version of this stuff, in canvas_course_admin_tools, didn't make have lti_school_permissions, manage_people, or manage_sections. Without them in INSTALLED_APPS, Django fails to process canvas_manage_course/urls.py when a command is run under these settings. (Not sure why `async` was also causing a failure here, possibly because of a move from Django 1.8 to 1.9?) If there is a more elegant way to avoid these apps that aren't germane to a command, would be good to know.